src/jarabe/journal/projectview.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2016, Abhijit Patel
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+import logging
+import dbus
+
+from gettext import gettext as _
+
+from gi.repository import GObject
+from gi.repository import Gtk
+
+from jarabe.view.friendlistpopup import FriendListPopup
+from jarabe.journal.expandedentry import TextView, BuddyList
+from jarabe.journal.detailview import BackBar
+from jarabe.journal.listview import ListView
+from jarabe.journal import model
+
+from sugar3.graphics import style
+from sugar3.graphics.toolbutton import ToolButton
+
+_SERVICE_NAME = 'org.laptop.Activity'
+_SERVICE_PATH = '/org/laptop/Activity'
+_SERVICE_INTERFACE = 'org.laptop.Activity'
+
+class ProjectView(Gtk.VBox):
```

Comment:
Can you reuse some code from the DetailsView here?

I would love for the "top" of the project view (with the title and description box) to look *exactly* the same as the "top" of the details view.  Consistency is great.

So the top of the project view should have the same title entry box like this:

![screenshot from 2016-07-09 14-42-02](https://cloud.githubusercontent.com/assets/6022042/16706123/7ee74eca-458f-11e6-8ba1-8a97945f9d48.png)


and the same description textbox/label style.